### DISCUSSION TITLE
reader mode, no preview

### DISCUSSION THREAD

**Discussion by mexsudo** [lwindolf/liferea#1363] [Q&A]
Debian Stable, Mate
the "preview" is blank in reader mode and also when I in browser
I still have the ability to open in external browser. 
![Liferea 2024-05-15 08-49-26](https://github.com/lwindolf/liferea/assets/7226539/31e25baf-68f0-4ea9-93ec-6bf7dbf9dd7c)

**Comment by mexsudo** [comment]
1. same machine, booting with Debian Sid, Mate
same problem.
2. Different machine, Debian Mate. fresh install of Liferea... same peoblem

**Reply by lwindolf** [reply]
If you have an NVidia graphics card you might be affected by https://github.com/lwindolf/liferea/issues/1308 which is not caused by Liferea and has a workaround.

You can check this by using GNOME web (epiphany) to load a page. Does it have the same problem?

**Reply by mexsudo** [reply]
Yes I have NVIDIA GK106GL [Quadro K4000]
workarounds are no help...

I did not try the Gnome thing

**Reply by lwindolf** [reply]
@mexsudo Sorry to hear this. The issue described above was experienced by different other NVidia graphic card owners who all could fix it. I'm 100% certain that it is cause by your graphic card.
